lnrpc/routerrpc/router_server.go

```diff
@@ -1133,15 +1133,11 @@ func toPairSnapshot(pairResult *PairHistory) (*routing.MissionControlPairSnapsho
 			"differ", pairPrefix)
 	}
 
-	failAmt, failTime, err := getPair(
+	failAmt, failTime, _ := getPair(
```

Comment:
I don't think ignoring the error is a good idea. Maybe we can pass in a `isFailure` boolean into `getPair()` and then a zero-amount isn't seen as an error?